app/assets/javascripts/discourse/widgets/user-menu.js.es6

```diff
@@ -81,21 +85,149 @@ createWidget('user-menu-links', {
   }
 });
 
+export default createWidget('user-bookmarks', {
+  buildKey: () => 'user-bookmarks',
+
+  defaultState() {
+    return { loading: false, loaded: false, content: [] };
+  },
+
+  refreshBookmarks(state) {
+    if (state.loading) { return; }
+
+    const { currentUser } = this;
+    let findUrl = Discourse.getURL('/user_actions.json?username=' + currentUser.username + "&filter=" + Discourse.UserAction.TYPES.bookmarks);
```

Comment:
You can just pass the path into `ajax` which [will call getURL](https://github.com/discourse/discourse/blob/master/app/assets/javascripts/discourse/lib/ajax.js.es6#L103) for you